.github/workflows/test.yml

```diff
@@ -39,15 +39,17 @@ jobs:
           - { ruby: '3.3', gemfile: 'rails_7_0' }
```

Comment:
Can we remove testing for ruby 3.3 on rails 7.0 and 7.1 (As per documentation above)